demos/python_demos/object_detection_demo_ssd_async/models.lst

```diff
@@ -1,11 +1,13 @@
 # This file can be used with the --list option of the model downloader.
+face-detection-????
 face-detection-adas-????
 face-detection-adas-binary-????
 face-detection-retail-????
 pedestrian-and-vehicle-detector-adas-????
 pedestrian-detection-adas-????
 pedestrian-detection-adas-binary-????
 pelee-coco
+person-detection-asl-????
```

Comment:
Your PR description says that this also adds support for `person-detection-????`, so shouldn't it also be added?

Reply:
Yes, thanks. I removed those for a while. As long as they are available now, readded them to the list.